### PR TITLE
Ankit | Test  | Version 3 - Whatsapp support for vouchers with default template

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cw8m5eu

### CODE REVIEW

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.ts [hunk 1]
```diff
@@ -282,4 +282,16 @@ export class PurchaseSettingComponent implements OnInit, OnDestroy {
...
+
+    /**
+     * Send voucher type whatsapp option.
+     *
+     * @param {string} voucherType
+     * @returns {boolean}
+     * @memberof PurchaseSettingComponent
+     */
+    getWhatsappOption(voucherType: string): string {
+        let text = this.commonLocaleData?.app_send_voucher_type_whatsapp?.replace("[VOUCHER_TYPE]", voucherType);
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.ts]:
```suggestion
        return this.commonLocaleData?.app_send_voucher_type_whatsapp?.replace("[VOUCHER_TYPE]", voucherType);
```

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.ts [hunk 2]
```diff
@@ -282,4 +282,16 @@ export class PurchaseSettingComponent implements OnInit, OnDestroy {
...
     }
+
+    /**
+     * Send voucher type whatsapp option.
+     *
+     * @param {string} voucherType
+     * @returns {boolean}
+     * @memberof PurchaseSettingComponent
+     */
+    getWhatsappOption(voucherType: string): string {
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.ts]:
```suggestion
    public getWhatsappSettingLabelvoucherType: string): string {
```

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html [hunk 3]
```diff
@@ -385,6 +385,56 @@ <h4 class="clearfix bd-rb">{{ localeData?.purchase_bill }}</h4>
                                         </div>
                                     </div>
                                 </div>
+                                <div class="clearfix pd-t1" *ngIf="voucherApiVersion === 2">
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html]: *ngIf="voucherApiVersion === 2"

Write this ng-container which will common for all below settings

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.ts (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.ts]: Done

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.ts (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.ts]: Done

File: apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html (hunk 3 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/purchase/purchase-setting/purchase-setting.component.html]: Done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**